core/controllers/platform_feature_test.py

```diff
@@ -34,7 +34,7 @@ class ParamNames(enum.Enum):
     PARAMETER_B = 'parameter_b'
 
 
-DATA_TYPES = param_domain.DATA_TYPES # pylint: disable=invalid-name
+DataTypes = param_domain.DataTypes
```

Comment:
I don't think we even need to create this variable, just use `param_domain.DataTypes` directly down below.